apps/member/src/components/modal/CheckConfirmModal/CheckConfirmModal.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button } from '@clab-platforms/design-system';
+
+import Modal from '@components/common/Modal/Modal';
+
+interface CheckConfirmModalProps {
+  message: string;
+  handleConfirmButton: () => void;
+  handleClose: () => void;
+}
+
+const CheckConfirmModal = ({
+  message,
+  handleConfirmButton,
+  handleClose,
+}: CheckConfirmModalProps) => {
+  return (
+    <Modal>
+      <Modal.Header>확인</Modal.Header>
+      <Modal.Body>
+        <p>{message}</p>
+      </Modal.Body>
+      <Modal.Footer className="flex space-x-2">
+        <Button
+          className="w-full"
+          color="blue"
+          onClick={() => {
+            handleConfirmButton();
+            handleClose();
+          }}
+        >
+          확인
+        </Button>
+        <Button className="w-full" color="red" onClick={() => handleClose()}>
```

Comment:
인자가 필요한게 아니라면 굳이 화살함수로 표기할 필요는 없어보여요

```suggestion
        <Button className="w-full" color="red" onClick={handleClose}>
```

Reply:
헉 코드 변경할 때 수정이 안 됐나봐요 알려주셔서 감사해요 :)